cfi_backward.adoc

```diff
@@ -228,6 +228,10 @@ else
 endif
 ----
 
+The `ssp` is incremented by `sspop` only if the load from shadow stack completes
```

Comment:
`sspopchkra` need similar wordings?

The `ssp` is incremented by `sspopchkra` only if the load from shadow stack completes successfully.

Reply:
Please see PR #59

Reply:
great